src/containers/CommunityProjects/CommunityProjectsInfo.tsx

```diff
@@ -0,0 +1,35 @@
+export const projInfo = [
```

Comment:
nit: prefer non abbreviations.. few extra characters but helps code readability 
```suggestion
export const projectInfo = [
```

Reply:
👍 bd33acc